service/framework.go

```diff
@@ -54,9 +51,9 @@ func newCRDFramework(config Config) (*framework.Framework, error) {
 
 	var err error
 
-	var k8sExtClient apiextensionsclient.Interface
+	var restConfig *rest.Config
```

Comment:
This should be the state of the art for creating the framework. But would appreciate a 2nd opinion.